src/wp-admin/includes/upgrade.php

```diff
@@ -2275,6 +2279,35 @@ function upgrade_560() {
 		save_mod_rewrite_rules();
 	}
 }
+/**
+ * Executes changes made in WordPress 5.7.0.
+ *
+ * @ignore
+ * @since 5.7.0
+ */
+function upgrade_570() {
+	global $wp_current_db_version, $wpdb;
+
+	if ( $wp_current_db_version < 51769 ) {
+		/*
+		 * Create fulltext indexes for `wp_posts`.
+		 */
+		$result = $wpdb->query( "ALTER TABLE $wpdb->posts ADD FULLTEXT INDEX `wp_posts_fulltext` (`post_title` ASC, `post_excerpt` ASC, `post_content` ASC);" );
+		// Return early if fulltext indexes are unsupported
+		if ( ! $result || is_wp_error( $result ) ) {
+			return;
```

Comment:
This is not graceful as any errors will be displayed to the user. I welcome all the suggestions on how to silence the errors here.